### ISSUE TITLE
Accessibility - "Jump to" menu on section page needs label.

### ISSUE THREAD

**Issue by stopfstedt** [gjbarnard/moodle-format_topcoll#183]
**Describe the bug**
This is an accessibility (a11y) violation, see https://dequeuniversity.com/rules/axe/4.10/select-name?application=AxeFirefox for details.

**To Reproduce**
Steps to reproduce the behaviour:
1. Log in as user
2. Create a course, select "Collapsed Topics" as format. Click "Save and display".
3. Click the "Topic 1" link next to the section header to only view that section.
4. At the bottom of the page, observe that there is a dropdown menu with the "Jump to..." option selected.
5. Open your browser's dev tools/inspector and find that dropdown by using the `select.jumpmenu` query selector.
6. Observe that that dropdown `<select>` element is unlabeled. 

**Expected behaviour**
There should be a text label on that dropdown.

**Screenshots**

<img width="1290" height="838" alt="Image" src="https://github.com/user-attachments/assets/17f56c38-5a0d-4d06-ab92-b1b12368db76" />

**Versions (please complete the following information):**
 - Moodle: 4.5,  5.0, 5.1, main
 - Format: V500.1.3,  V405.1.4
 - Browser and version: doesn't really matter here. I'm on latest FF/Ubuntu.

**Additional Context**
You can hit that a11y violation by scanning the section page using one of the WAVE browser extensions. https://wave.webaim.org/extension/ 

<img width="1664" height="768" alt="Image" src="https://github.com/user-attachments/assets/41c37228-59af-463b-a1fd-998996086139" />

**Comment by stopfstedt** [comment]
PRs with fixes incoming shortly.

**Comment by gjb2048** [comment]
Dear @stopfstedt,

Thanks for the information and patches.  This is the release repository, so I'll patch and put in the next version if the changes pass testing.

Kind regards,

Gareth

**Comment by stopfstedt** [comment]
Thanks Gareth.
Btw, I just force-pushed a change on this [PR](https://github.com/gjbarnard/moodle-format_topcoll/pull/184) to realign them - I had a minor deviation that in that one where I was using a deprecated CSS class.
Should be all ready for review now.
